CreateMessage/handler.ts

```diff
@@ -259,28 +298,43 @@ const redirectToNewMessage = (
     { id: newMessageWithoutContent.id }
   );
 
+/*
+ ** The right full path for ownerID is in this kind of format:
+ ** "/subscriptions/subid/resourceGroups/{resourceGroup}/providers/Microsoft.ApiManagement/service/{apimService}/users/5931a75ae4bbd512a88c680b",
+ ** resouce link: https://docs.microsoft.com/en-us/rest/api/apimanagement/current-ga/subscription/get
+ */
+export const parseOwnerIdFullPath = (
+  fullPath: NonEmptyString
+): NonEmptyString =>
+  pipe(
+    fullPath,
+    f => f.split("/"),
+    a => a[a.length - 1] as NonEmptyString
+  );
```

Comment:
1. what if `a[index]` does not exists?
2. we should avoid to use `as`, the only way to make sure that the return type is `NonEmptyString` we have to parse with `NonEmptyString.decode`

Reply:
For the first point:
We are using the `split` method on a `NonEmptyString` so i think that the scenario of a non existing `a[index]` will never happen because the split method should always return an array with at least one element.
We can add a check on the length before the split to be sure tho

Reply:
As garma said, split always return an array. In fact that code is working also when we pass a simple user id without full path. Splitting by slash gives us an array with an only word inside.